### ISSUE TITLE
Entities Don't show up in Amulet Editor

### ISSUE THREAD

**Issue by AndrewBraunEE** [Amulet-Team/Amulet-Map-Editor#171]
**Has this bug already been reported? Please check if there is a bug report already open that matches this.**

**Describe the bug**
Entities (eg mobs, minecarts, etc. don't show up in Amulet Editor).

**To Reproduce**

**Expected behavior**
Entities would have some sort of visual during editing.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: Beta (Latest)

**Comment by SteamDoge9000** [comment]
wow, I hope this does get added soon, the MCedit 2.0 has this but for some reason cant get it to load  in 1.16.5 without it crashing

I was going to ask on the discord but found my answer on fthe FAQ page. I wish you all good luck ^w^

**Comment by DrakorthaP** [comment]
Entities such as animals and boats do not copy and paste either.

**Comment by DrakorthaP** [comment]
could this issue be made higher priority ?

**Comment by HazeTheHawk13** [comment]
I really wish this was high priority

**Comment by FrankTwenty1** [comment]
I think a higher priority is important too!

The entities do not have to be visible in the editor at first, but they should at least be copied.

**Comment by Itzikr1** [comment]
Would also love this, to be moved higher in the priority

**Comment by MavisCelus19201** [comment]
Yeah this could definitively use the high priority tag.

**Comment by Jelte2357** [comment]
I Need this too! this is the only real option if you want to edit things in bedrock worlds, if you dont count paid programs.

**Comment by J-Freak** [comment]
Agreed, I'd love if this were a higher priority! :) I've found another group of friends who play on another server and we want to put both our builds from different worlds into one world, but without this feature it means one world will need to accept losing all their pets, villagers, nametagged mobs, and hopper minecarts involved in farms etc

**Comment by pointydev** [comment]
Still would love to see this added, for my use case the editor is virtually pointless without it.

**Comment by kohlrak** [comment]
At first glance this seems like low-hanging fruit compared to the other things on the list. I'm curious what exactly the road block is with this in particular.

**Comment by gentlegiantJGC** [comment]
The translation engine has been standing in the way. I have been rewriting Amulet so that the translator is less integrated into its function. That way the raw data can be directly accessed.

**Comment by kohlrak** [comment]
Awesome, it's good to hear that this still a priority. I found another that does do the conversion, entities and all, but seems to be missing sign text, and i'm not sure if it's only the front end that's open source or the back end as well (not a big deal thugh). If you need any help, i'd offer it, but i haven't used java with any seriousness since 2007 or 2008, and never for minecraft. This is certainly one of the things the minecraft community needs, for both directions. The disparities between the two versions are great, but the most important thing has always been translating worlds without loosing what is important, and of course mobs are one of the biggest things in any survival world translation.

**Comment by J-Freak** [comment]
What was the name of the program that is able to move entities? I've had villager trading halls of over 100 villagers I've had to start on from scratch 😅

**Comment by kohlrak** [comment]
> What was the name of the program that is able to move entities? I've had villager trading halls of over 100 villagers I've had to start on from scratch 😅

je2be, don't let the name fool you. It's not perfect, either, and i just had my first issue with it tonight: failed to download my larger world. Not sure what the problem is and i'm trying again, but it seems to be preserving trades too. For the time being, it's something. Just be aware, obviously, ownership of pets is going to be a problem: they're still owned, but there's a chance you won't be the owner (i'm still working on it and some other things so you might be, based on my brand new usage of geyser giving me access to my java inventory telling me i probably have the same uuid which is what affects it) and have to use some console commands to get ownership. That said, when amulet is ready to go, that'll be a whole lot easier.
